app/models/entry.rb

```diff
@@ -1,3 +1,42 @@
 class Entry < ApplicationRecord
   belongs_to :basket
+
+  EXEMPT_FROM_TAX = %w[chocolate book pills].freeze
+  BASIC_TAX_RATE = 0.1
+  IMPORT_TAX_RATE = 0.05
+  PRESITION = 20.0
```

Comment:
Nice use of variables 👏🏽 👏🏽 

Small convention: constants should go above macro methods such as `belong_to`